example.yml

```diff
@@ -82,6 +82,7 @@ modules:
         - expect: "^:[^ ]+ 001"
   icmp_example:
     prober: icmp
+    source_address: "127.0.0.1"
```

Comment:
it's odd that this is in the probe but preferred_ip_protocol is in the module

Reply:
True. But your "This should apply to all probes" made me agree and also implement source-address specification for the other probers. Is it possible (from a configuration compatibility perspective) to pull preferred_ip_protocol up to this level as well?

Reply:
That'd be a breaking change, so best to keep it consistent.

Reply:
So you propose to move it back into the prober-specific configuration?

Reply:
Yes